src/lib/booleans.js

```diff
@@ -0,0 +1,131 @@
+function negate(a) {
+  // return !a - to return opposite of a
+  if (a === true) {
+    return false;
+  }
+  return true;
+};
+
+function both(a, b) {
+  if (a && b === true) {
+    return true;
+  }
+  return false;
+}
+
+function either(a, b) {
+  if (a || b === true) {
+    return true;
+  }
+  return false;
+};
+
+function none(a, b) {
+  if (a === false && b === false){
+    return true;
+  }
+  return false;
+};
+
+function one(a, b) {
+  if (a === true && b === false) {
+    return true;
+  }
+  if (a === false && b === true) {
+    return true;
+  }
+  return false;
+};
+
+function truthiness(a) {
+  if (a) {
+    return true;
+  }
+  return false;
+};
+
+function isEqual(a, b) {
+  if (a === b) {
+    return true;
+  }
+  return false;
+};
+
+function isGreaterThan(a, b) {
+  if (a > b){
+    return true;
+  }
+  return false;
+};
+
+function isLessThanOrEqualTo(a, b) {
+  if (a <= b){
+    return true;
+  }
+  return false;
+};
+
+function isOdd(a) {
+  if (a % 2 === 0){
+    return false;
+  }
+  return true;
+};
+
+function isEven(a) {
+  if (a % 2 === 0){
+    return true;
+  }
+  return false;
+};
+
+function isSquare(a) {
+  if (Math.sqrt(a) % 1 === 0){
+    return true;
+  }
+  return false;
+};
```

Comment:
All these could be reduced in the same way as detailed above. When you are looking to return a true and false value dependent on the evaluation of an expression, you can just return the evaluation of the expression as in the examples above. Makes it much cleaner and easier straight off :)